snet_cli/commands.py

```diff
@@ -504,3 +510,225 @@ def transact(self):
             return receipt, events
         else:
             self._error("Cancelled")
+
+
+class ServiceCommand(BlockchainCommand):
+
+    def _tar_imports(self, paths):
```

Comment:
still needs a comment, unclear what's going on with the sorting and accounting of how deep paths are etc.